src/plots/cartesian/graph_interact.js

```diff
@@ -84,10 +84,6 @@ module.exports = function initInteractions(gd) {
                 dragElement.unhover(gd, evt);
             };
 
-            maindrag.onclick = function(evt) {
-                Fx.click(gd, evt, subplot);
```

Comment:
this is now defined in `cartesian/dragbox.js` correct?

Reply:
> this is now defined in `cartesian/dragbox.js`, correct?

yep, now it's here: https://github.com/plotly/plotly.js/pull/2241/files#diff-1bd3dd52cddb8638876bbe17cef28a0dR175